lib/utils/dialect.ts

```diff
@@ -0,0 +1,69 @@
+import isPlainObject from 'lodash/isPlainObject';
+// eslint-disable-next-line import/order
+import { v1 as uuidv1, v4 as uuidv4 } from 'uuid';
+
+// eslint-disable-next-line @typescript-eslint/no-require-imports
+const DataTypes = require('../data-types');
+
+const dialects = new Set([
+  'mariadb',
+  'mysql',
+  'postgres',
+  'sqlite',
+  'mssql',
+  'db2',
+]);
+
+export function now(dialect: string): Date {
+  const d = new Date();
+  if (!dialects.has(dialect)) {
+    d.setMilliseconds(0);
+  }
+
+  return d;
+}
+
+export function toDefaultValue(value: unknown, dialect: string): unknown {
+  if (typeof value === 'function') {
+    const tmp = value();
+    if (tmp instanceof DataTypes.ABSTRACT) {
+      return tmp.toSql();
+    }
+
+    return tmp;
+  }
+
+  if (value instanceof DataTypes.UUIDV1) {
+    return uuidv1();
+  }
+
+  if (value instanceof DataTypes.UUIDV4) {
+    return uuidv4();
+  }
+
+  if (value instanceof DataTypes.NOW) {
+    return now(dialect);
+  }
+
+  if (Array.isArray(value)) {
+    return [...value];
+  }
+
+  if (isPlainObject(value)) {
+    return { ...(value as object) };
+  }
+
+  return value;
+}
+
+// Note: Use the `quoteIdentifier()` and `escape()` methods on the
```

Comment:
For who is this note?

Reply:
I can't say. I think it's just a warning to maintainers that may try escaping using that variable?

Reply:
Ah, this was already present in the JS file?

Reply:
Yep!